spicy/toolchain/include/compiler/detail/codegen/codegen.h

```diff
@@ -52,6 +54,7 @@ class CodeGen {
     NodeRef preserveNode(Expression x);
     NodeRef preserveNode(Statement x);
     NodeRef preserveNode(Type x);
+    auto uniquer() { return &_uniquer; }
```

Comment:
Not a real issue here, but it would be nice if this method could return a `const` pointer instead to be more in line with the overall spirit of this class. That would probably require some changes to `Cache` so it can provide a `const` interface over internally `mutable` state.

Feel free to ignore.